assets/js/script.js

```diff
@@ -46,14 +49,62 @@ jQuery( function ( $ ) {
 			const value = $( this ).val().toLowerCase();
 
 			$( '.file-upload-types-table table tr' ).filter( function () {
-				if ( ! $( this ).hasClass( 'heading' ) ) {
-					if ( value !== '' && $( this ).hasClass( 'section' ) ) {
-						$( this ).hide();
+
+				const $this = $( this );
+
+				if ( ! $this.hasClass( 'heading' ) ) {
+					if ( value !== '' && $this.hasClass( 'section' ) ) {
+						$this.hide();
 					}
-					else {
-						$( this ).toggle( $( this ).text().toLowerCase().indexOf( value ) > - 1 );
+					// Do nothing of this is hidden repetitive section with no value yet.
+					else if ( $this.hasClass( 'repetitive-fields' ) && $this.css( 'display' ) === 'none' && ! $this.find( 'input' ).toArray().some( input => input.value !== '' ) ) {
+						return;
+					} else {
+						$this.toggle( $this.text().toLowerCase().indexOf( value ) > - 1 );
 					}
 				}
 			} );
+		} )
+		.on( 'click', '#c_types_file_sample_button a', function ( e ) {
+			e.preventDefault();
+
+			$( '.repetitive-fields' ).show();
 		} );
+
+	let uploaded = 0;
+
+	let SampleFileDropzone = new Dropzone( '#c_types_file_sample_button', {
+		url: ajaxurl,
```

Comment:
We can add the `ajaxurl` in the `global` comment on the 1st line of the file
or 
We can add the `ajaxurl` to our `file_upload_types_params` variable.

Reply:
done the first one